public_html/layouts/resources/Fields.js

```diff
@@ -1478,18 +1479,20 @@ window.App.Fields = {
 		 */
 		toggleCheckBox(element) {
 			if ($(element).is(':checked')) {
+				$(element).attr('checked', 'checked');
 				element
 					.closest('label.js-label-checkbox')
 					.eq(0)
-					.find('svg.svg-inline--fa')
+					.find('span.far')
```

Comment:
We avoid references to html tags, instead add class with js- prefix for reference in js

Reply:
I will try to keep it in mind :D